packages/gatsby/src/utils/__tests__/cache-lmdb.ts

```diff
@@ -0,0 +1,47 @@
+import * as path from "path"
+import fs from "fs-extra"
+import { describeWhenLMDB } from "../worker/__tests__/test-helpers"
+
+const complexObject = {
+  key: `value`,
+  another_key: 2,
+  nested: { hello: `world`, foo: `bar`, nested: { super: `duper` } },
+}
+
+describeWhenLMDB(`cache-lmdb`, () => {
+  let cache
+
+  beforeAll(async () => {
+    const { default: GatsbyCacheLmdb } = await import(`../cache-lmdb`)
+    cache = new GatsbyCacheLmdb({ name: `__test__` })
+    const fileDir = path.join(
+      process.cwd(),
+      `.cache/caches-lmdb-${process.env.JEST_WORKER_ID}`
```

Comment:
Curious why you need to postfix it with worker id? Jest runs tests on multiple workers per file not test.

Reply:
Mostly did that because that's how the class uses itself: https://github.com/gatsbyjs/gatsby/pull/32373/files#diff-efecbacd93ffd4230b01b2b38d8e9ab6b06f723b31146e8a3f08e747123a885aR11

Reply:
I think this just clears the folder with the previous version of db. The actual path is set in `cache-lmdb.ts` and we need it in case if multiple tests use cache.

Reply:
That's a little bit weird but I don't have enough context so 👍

Reply:
This is not for this test alone. Other tests running in other processes may be writing/cleaning the same db.